src/components/display/edit/EditDisplay.js

```diff
@@ -118,6 +139,10 @@ export class EditDisplay extends Component {
                 <EditToolbar
                     path={path}
                     handleSave={this.handleSaveValue.bind(this)}
+                    handleFormat={this.handleFormatValue.bind(this)}
+                    handleFormatCompact={this.handleFormatValueCompact.bind(
+                        this
+                    )}
```

Comment:
MINOR: Seems a bit cleaner to do one of the following (also for `handleSave`):
- Bind to this in constructor
- Use class property arrow functions

Reply:
I've moved them to the constructor as that seems to be the approach elsewhere in the app.